txscript/data/script_tests.json

```diff
@@ -2242,14 +2277,7 @@
 ["2147483648 1", "BOOLOR 1", "NONE", "ERR_OUT_OF_RANGE", "We cannot do BOOLOR on 5-byte integers (but we can still do IF etc)"],
 ["2147483648 1", "BOOLAND 1", "NONE", "ERR_OUT_OF_RANGE", "We cannot do BOOLAND on 5-byte integers"],
 
-["1", "1 ENDIF", "NONE", "ERR_UNBALANCED_CONDITIONAL", "ENDIF without IF"],
-["1", "IF 1", "NONE", "ERR_UNBALANCED_CONDITIONAL", "IF without ENDIF"],
-["1 IF 1", "ENDIF", "NONE", "ERR_UNBALANCED_CONDITIONAL", "IFs don't carry over"],
-
-["NOP", "IF 1 ENDIF", "NONE", "ERR_INVALID_STACK_OPERATION", "The following tests check the if(stack.size() < N) tests in each opcode"],
-["NOP", "NOTIF 1 ENDIF", "NONE", "ERR_INVALID_STACK_OPERATION", "They are here to catch copy-and-paste errors"],
-["NOP", "VERIFY 1", "NONE", "ERR_INVALID_STACK_OPERATION", "Most of them are duplicated elsewhere,"],
-["NOP", "TOALTSTACK 1", "NONE", "ERR_INVALID_STACK_OPERATION", "but, hey, more is always better, right?"],
+["NOP", "TOALTSTACK 1", "NONE", "ERR_INVALID_STACK_OPERATION"],
```

Comment:
Since this test was added in this PR it needs a comment for what is being tested.